zeebe/exporters/opensearch-exporter/src/main/java/io/camunda/zeebe/exporter/opensearch/OpensearchExporterConfiguration.java

```diff
@@ -225,6 +225,7 @@ public static class IndexConfiguration {
     public boolean conditionalEvaluation = false;
 
     public boolean globalListenerBatch = false;
+    public boolean expression = false;
```

Comment:
❓ The same question as with the elasticsearch-exporter. However, if this is required, it seems that the required changes are not fully implemented.